src/libraries/System.IO.FileSystem/src/System/IO/FileSystem.Unix.cs

```diff
@@ -149,16 +149,6 @@ public static void MoveFile(string sourceFullPath, string destFullPath, bool ove
             // changing the casing in the naming of the file. If this fails in any way (e.g. source file
             // doesn't exist, dest file doesn't exist, rename fails, etc.), we just fall back to trying the
             // link/unlink approach and generating any exceptional messages from there as necessary.
```

Comment:
This large comment should be moved as well.

Reply:
Agree, moved.